tests/pipelines/test_split_step.py

```diff
@@ -128,6 +128,7 @@ def test_from_pipeline_config_works_with_target_col(tmp_path):
         assert SplitStep.from_pipeline_config({"target_col": "fake_col"}, "fake_root") is not None
 
 
+@pytest.mark.mlp_next_release("1.29.0")
```

Comment:
Adding this decorator as a test. Because MLflow's dev version is currently 1.28.0.dev0, only this test case should run when `pytest --mlp-next-release` is invoked. Additionally, all tests except this one should run when `pytest` is invoked *without* the `--mlp-next-release` flag.

@dbczumar Remove this before merging.